### PR TITLE
readded plugin.properties. fixes #154

### LINKED ISSUE #154: Version recognition does not work
I tested the wizard as installed from the Jenkins update sites. The resulting build.gradle did not contain the correct version `2.11.0-SNAPSHOT`, but just `unknown`.

### PR DESCRIPTION
**Pull request by cdietrich**
readded plugin.properties. fixes #154

Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

**Comment by cdietrich** [comment]
please review

File: org.eclipse.xtext.util/build.gradle [hunk 1]
```diff
@@ -5,3 +5,9 @@ dependencies {
 	compile "org.eclipse.emf:org.eclipse.emf.ecore:$versions.emfEcore"
 	compile "com.google.inject:guice:$versions.guice"
 }
+
+jar {
+	from('.') {
+		include 'plugin.properties'
```
Comment (spoenemann) [inline org.eclipse.xtext.util/build.gradle]: Please move this configuration to `eclipse-project-layout.gradle`. If a plugin.properties is present, it should always be included in the build.

**Comment by spoenemann** [comment]
The new plugin.properties is empty? What is it needed for?

**Comment by cdietrich** [comment]
this is the way we read the version from manifest

- org.eclipse.xtext.util.XtextVersion.readVersionFromManifest() calls (indirectly)
- org.eclipse.emf.common.util.DelegatingResourceLocator.getBaseURL()
- crashes when there is no plugin.properties

**Comment by spoenemann** [comment]
Then please add a comment in plugin.properties that explains why it must be there.

**Pull request merged**